pkg/agent/datamodel/sig_config.go

```diff
@@ -204,8 +205,10 @@ const (
 )
 
 const (
-	LinuxSIGImageVersion   string = "2022.03.15"
-	WindowsSIGImageVersion string = "17763.2686.220317"
+	LinuxSIGImageVersion string = "2022.03.15"
+
+	Windows2019SIGImageVersion string = "17763.2686.220317"
+	Windows2022SIGImageVersion string = "20348.524.220218"
```

Comment:
Do you plan to publish the first WS2022 in all regions before merging the code in aks rp?

Reply:
Yes